src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -406,6 +406,13 @@ export default class OrgPerson extends Mixins(CommonMixin) {
     return (this.orgPerson?.officer.partyType === PartyTypes.PERSON)
   }
 
+  /** Text same target */
+  get orgTargetType (): string {
+    let target = 'business'
+    if (this.isProprietor) target = this.isPerson ? 'person' : 'business'
+    return target
```

Comment:
Can you simplify this please?

Reply:
There are three cases to consider here. If it is SP, then we look for the whether it is for person or not. If it is GP, then the string is 'business'. The following is from Yui ([11932](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/11932)):

Note that in SP, there are two cases. Case A: the proprietor is an individual and we want "person" in the sentence. Case B: the proprietor is a business and we want "business" in the sentence.

Reply:
The logic is complicated to read/review. It's hard to tell there are 3 cases and whether or not the initial value is overridden.

The code above features a single return statement, but I'd prefer multiple return statements if it simplifies the code.

Reply:
Thanks, the new code is much easier to understand.